<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWeekDisplaytextDlg</class>
 <widget class="QWidget" name="QWeekDisplaytextDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="m_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>S#</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文本</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>191</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>属性</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>81</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>字体大小：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_fontsizecmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>113</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜    色：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>173</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>字    距：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_wordintervalspin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>171</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>位置</string>
    </property>
    <widget class="QRadioButton" name="m_radio2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>16</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radio3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>18</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radio1">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>18</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radio5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>38</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radio6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radio4">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radio8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radio9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>62</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radio7">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>62</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Text_Color">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>语    言：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_languagecmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>144</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>闪    烁：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_flickcheck">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>否</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>是 (颜色切换)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>是 (文字显隐)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>字    体：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_fontcmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="m_textedit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>所有语言都使用第一种语言的文本</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>所有状态属性都和状态0相同</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_table</tabstop>
  <tabstop>m_languagecmb</tabstop>
  <tabstop>m_fontcmb</tabstop>
  <tabstop>m_fontsizecmb</tabstop>
  <tabstop>m_flickcheck</tabstop>
  <tabstop>m_wordintervalspin</tabstop>
  <tabstop>m_radio1</tabstop>
  <tabstop>m_radio2</tabstop>
  <tabstop>m_radio3</tabstop>
  <tabstop>m_radio4</tabstop>
  <tabstop>m_radio5</tabstop>
  <tabstop>m_radio6</tabstop>
  <tabstop>m_radio7</tabstop>
  <tabstop>m_radio8</tabstop>
  <tabstop>m_radio9</tabstop>
  <tabstop>m_textedit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
